solidity/deploy/02_deploy_vending_machine.ts

```diff
@@ -0,0 +1,24 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre
+  const { deploy } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  const TBTCToken = await deployments.get("TBTCToken")
+  const TBTC = await deployments.get("TBTC") // tBTC v2
+
+  const unmintFee = 1000000000000000 // 0.001 of the amount being unminted
```

Comment:
Why 0.001? Should be 0 at the beginning.

Reply:
All post-merge remarks addressed in https://github.com/keep-network/tbtc-v2/pull/27